files/assets/js/volunteer_exit.js

```diff
@@ -0,0 +1,16 @@
+// a tiny script to yell at the user if they try to leave the Volunteer page any way besides the Submit button
```

Comment:
minor thing: how about renaming to volunteer.js (so if ever future JS is needed, it can be added here to cut down on requests)